nncf/experimental/openvino_native/statistics/aggregator.py

```diff
@@ -69,7 +70,11 @@ def _register_statistics(self,
                     self._register_activation_statistic(statistic_point,
                                                         target_point, node_name, outputs)
                 elif target_point.type == TargetType.OPERATION_WITH_WEIGHTS:
-                    self._register_weight_statistic(statistic_point, target_point)
+                    # Register constant only once because it does not change
```

Comment:
Are that changes for the shared weights? Is there any test for such a case added?

Reply:
This optimization forces to register statistics for weights only ones, regardless whether it  is shared or not. `test_statistics_aggregator` checks weight statistic registered at least ones. 
Tests for shared weights are missing now in all backends, I suggest to add them in a different PR

Reply:
Ticket: 104570